examples/nlp-case/run.py

```diff
@@ -67,7 +67,7 @@
 @click.option(
     "--no-cache",
     is_flag=True,
-    default=False,
+    default=True,
```

Comment:
Was this intended or just a test leftover ?

Reply:
No, it is from Safoine's template - don't know the intention, but it was just auto-refreshed